src/pkg/packaging/rpm/scripts/after_remove_host.sh

```diff
@@ -3,5 +3,8 @@
 # Copyright (c) .NET Foundation and contributors. All rights reserved.
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 #
-echo "Removing dotnet host symbolic link"
-unlink /usr/bin/dotnet
+# Run the script only when newer version is not installed. Skip during package upgrade 
+if [ $1 = 0 ]; then
```

Comment:
(nit) I think it reads easier if you give a name to the arguments.  That way someone doesn't have to look up what the `$1` arg is.

See 

https://github.com/dotnet/core-setup/blob/eef6a7b1fe6883bee0aa967eb1a434c19f6990f4/src/pkg/packaging/osx/sharedhost/scripts/postinstall#L7-L8

As an example.

Reply:
This is by design. Fedora guidance for spec file specifies the variable as $1 . See https://fedoraproject.org/wiki/Packaging:Scriptlets